spec/controllers/results_controller_spec.rb

```diff
@@ -479,6 +479,93 @@ def self.test_unauthorized(route_name)
     end
   end
 
+  shared_examples 'showing json data' do |is_student|
+    let(:student2) do
+      partner = create(:student, grace_credits: 2)
+      StudentMembership.create(user: partner,
```

Comment:
This is fine for now but in the future, please use the factories when creating objects for tests:

```ruby
create :accepted_student_membership, user: partner, grouping: grouping
```